compiler/ballerina-lang/src/main/java/io/ballerina/projects/JBallerinaBackend.java

```diff
@@ -542,12 +545,21 @@ private Path emitGraalExecutable(Path executableFilePath) {
         String nativeImageName;
         String[] command;
         Project project = this.packageContext().project();
+        String nativeImageCommand = System.getenv("GRAALVM_HOME");
+
+        if (nativeImageCommand == null) {
+            throw new ProjectException("unable to create native image. The environment variable GRAALVM_HOME " +
```

Comment:
Can we change this error message like :
```
GraalVM installation directory not found. Set GRAALVM_HOME as an environment variable.
```

Reply:
Changed the wording of the exception error message